webclient/src/main.js

```diff
@@ -3,8 +3,10 @@
 import Vue from 'vue';
 import Hacksite from './Hacksite';
 import router from './router';
+import VeeValidate from 'vee-validate';
 
 Vue.config.productionTip = false;
+Vue.use(VeeValidate);
```

Comment:
This is also being done `router/index.js` can you make it so it's only doing it here?

Reply:
Done.